features/networking/metrics.feature

```diff
@@ -63,3 +63,58 @@ Feature: SDN/OVN metrics related networking scenarios
     And the output should contain:
       | kubeproxy_sync_proxy_rules_duration       |
       | kubeproxy_sync_proxy_rules_last_timestamp |
+  
+  # @author anusaxen@redhat.com
+  # @case_id OCP-37704
+  @admin
+  Scenario: Should be able to monitor various ovnkube-master and ovnkube-node metrics via prometheus
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-ovn-kubernetes" project
+    And evaluation of `endpoints('ovn-kubernetes-master').subsets.first.addresses.first.ip.to_s` is stored in the :ovn_master_metrics_ep_ip clipboard
+    And evaluation of `endpoints('ovn-kubernetes-master').subsets.first.ports.first.port.to_s` is stored in the :ovn_master_metrics_ep_port clipboard
+    And evaluation of `cb.ovn_master_metrics_ep_ip + ':' +cb.ovn_master_metrics_ep_port` is stored in the :ovn_master_metrics_ep clipboard
+    
+    And evaluation of `endpoints('ovn-kubernetes-node').subsets.first.addresses.first.ip.to_s` is stored in the :ovn_node_metrics_ep_ip clipboard
+    And evaluation of `endpoints('ovn-kubernetes-node').subsets.first.ports.first.port.to_s` is stored in the :ovn_node_metrics_ep_port clipboard
+    And evaluation of `cb.ovn_node_metrics_ep_ip + ':' +cb.ovn_node_metrics_ep_port` is stored in the :ovn_node_metrics_ep clipboard
+    
+    Given I use the "openshift-monitoring" project
+    And evaluation of `secret(service_account('prometheus-k8s').get_secret_names.find {|s| s.match('token')}).token` is stored in the :sa_token clipboard
+    
+    Given evaluation of `%Q{curl -k -H \"Authorization: Bearer <%= cb.sa_token %>\" https://<%= cb.ovn_master_metrics_ep %>/metrics}` is stored in the :curl_query_for_ovn_master clipboard
+    Given evaluation of `%Q{curl -k -H \"Authorization: Bearer <%= cb.sa_token %>\" https://<%= cb.ovn_node_metrics_ep %>/metrics}` is stored in the :curl_query_for_ovn_node clipboard
+    When I run the :exec admin command with:
+      | n                | openshift-monitoring                |
+      | pod              | prometheus-k8s-0                    |
+      | c                | prometheus                          |
+      | oc_opts_end      |                                     |
+      | exec_command     | bash                                |
+      | exec_command_arg | -c                                  |
+      | exec_command     | <%= cb.curl_query_for_ovn_master %> |
```

Comment:
shouldn't this be `exec_command_arg`?

Reply:
Ya ideally it should be ```exec_command_arg```. Taken care of. Apparently it was executing without any issues though. Thanks for catching that.
Ya the eval part is tricky specially when you have more quote/special characters in a string, so storing via ruby ```%Q``` in clipboard and calling it later, a neat way :) I left a comment there.
Re-ran the test, looks good.